### DISCUSSION TITLE
Site is broken

### DISCUSSION THREAD

**Discussion by knownasilya** [talentlessguy/tinyhttp#310] [General]
FYI. Bad Gateway

**Comment by v1rtl** [comment]
oopsie I must've forgot to launch the container on the server

will fix now

**Comment by v1rtl** [comment]
try now

**Comment by knownasilya** [comment]
Looks good

**Comment by go4cas** [comment]
Docs site throwing 502 (Bad Gateway) again.

**Comment by v1rtl** [comment]
does it happen still? i think imma move tinyhttp website off vultr to render
